### PR TITLE
edit cmd/cloud-controller-manager/app/nodeipamcontroller.go to struct…

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/cloud-controller-manager/app/nodeipamcontroller.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/cloud-controller-manager/nodeipamcontroller.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ type nodeIPAMController struct {
 func (nodeIpamController *nodeIPAMController) startNodeIpamControllerWrapper(completedConfig *cloudcontrollerconfig.CompletedConfig, cloud cloudprovider.Interface) app.InitFunc {
 	errors := nodeIpamController.nodeIPAMControllerOptions.Validate()
 	if len(errors) > 0 {
-		klog.Fatal("NodeIPAM controller values are not properly set.")
+		klog.ErrorS("NodeIPAM controller values are not properly set.")
```
Comment (cici37) [inline cmd/cloud-controller-manager/nodeipamcontroller.go:58]: Suggestion: 
klog.ErrorS(utilerrors.NewAggregate(errors), "NodeIPAM controller values are not properly set.")
OR
klog.ErrorS(nil, "NodeIPAM controller values are not properly set.")

Prefer the first one to keep the error details.

Comment (cici37) [reply cmd/cloud-controller-manager/nodeipamcontroller.go:58]: utilerrors import fyi:
utilerrors "k8s.io/apimachinery/pkg/util/errors"

Comment (cheftako) [reply cmd/cloud-controller-manager/nodeipamcontroller.go:58]: Not convinced that moving away from fatal is a good idea. I realize this is "sample" code but if the real version of this code has an error you have a non-functional cluster and your making it more difficult to debug.

File: cmd/cloud-controller-manager/nodeipamcontroller.go [hunk 2]
```diff
@@ -98,14 +98,14 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 	if len(strings.TrimSpace(nodeIPAMConfig.ServiceCIDR)) != 0 {
 		_, serviceCIDR, err = net.ParseCIDR(nodeIPAMConfig.ServiceCIDR)
 		if err != nil {
-			klog.Warningf("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
+			klog.InfoS("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
```
Comment (yangjunmyfm192085) [inline cmd/cloud-controller-manager/nodeipamcontroller.go:101]: Could you please use the right format?

**Comment by yangjunmyfm192085** [comment]
duplicated with https://github.com/kubernetes/kubernetes/pull/100672

**Comment by yangjunmyfm192085** [comment]
and duplicated with https://github.com/kubernetes/kubernetes/pull/99532

**Comment by pacoxu** [comment]
/lgtm cancel
/close 
for dup see https://github.com/kubernetes/kubernetes/pull/102469#issuecomment-853657109

**Pull request closed without merging**